src/config/constants.ts

```diff
@@ -66,6 +66,8 @@ export const RESUME_REQUEST_LABEL = 'Upload a resume';
 export const SKILLS_REQUEST_LABEL = 'What skills do you have?';
 export const SKILLS_REQUEST_PLACEHOLDER = 'Javascript, iOS, Android...';
 export const SCHOOL_REQUEST_LABEL = 'What school do you go to?';
+export const TRAVEL_REQUEST_LABEL =
```

Comment:
This sounds like everyone can get up to 100$ off of travel regardless from where

Reply:
I'm gonna update the labels in another PR, but the logic is that this is a reimbursement request of up to $100 and we'll review applications to determine how much they get based on their school location

Reply:
Have we considered using environment variables for these instead of a constants file? It would make for a more portable project and allow for easier tweaking of this stuff on the fly. 

https://docs.netlify.com/configure-builds/environment-variables/#declare-variables

<img width="937" alt="image" src="https://user-images.githubusercontent.com/2529120/70863996-fdbfb980-1f1b-11ea-8b43-983840f1b333.png">